frontend/src/utils/api.ts

```diff
@@ -99,7 +99,7 @@ const handleResponse = async (response: Response) => {
     } else {
       data = await response.blob();
     }
-    return { response: response, data: data };
+    return Promise.resolve({ response: response, data: data });
```

Comment:
What's the difference between wrapping a value in `Promise.resolve` and returning a value directly?

Reply:
I don't remember exactly why I did this now. I guess I thought if the else case was rejected then the if condition should probably resolve.